app/backend/app.py

```diff
@@ -205,15 +229,69 @@ def config():
             "showGPT4VOptions": current_app.config[CONFIG_GPT4V_DEPLOYED],
             "showSemanticRankerOption": current_app.config[CONFIG_SEMANTIC_RANKER_DEPLOYED],
             "showVectorOption": current_app.config[CONFIG_VECTOR_SEARCH_ENABLED],
+            "showUserUpload": current_app.config[CONFIG_USER_UPLOAD_ENABLED],
         }
     )
 
 
+@bp.post("/upload")
+@authenticated
+async def upload(auth_claims: dict[str, Any]):
+    request_files = await request.files
+    if "file" not in request_files:
+        # If no files were included in the request, return an error response
+        return jsonify({"message": "No file part in the request", "status": "failed"}), 400
+
+    user_oid = auth_claims["oid"]
+    file = request_files.getlist("file")[0]
+    user_blob_container_client: FileSystemClient = current_app.config[CONFIG_USER_BLOB_CONTAINER_CLIENT]
+    user_directory_client = user_blob_container_client.get_directory_client(user_oid)
+    await user_directory_client.set_access_control(owner=user_oid)
+    file_client = user_directory_client.get_file_client(file.filename)
+    file_io = file
+    file_io.name = file.filename
+    file_io = io.BufferedReader(file_io)
```

Comment:
There might be a simpler way to get the appropriate file type to pass into our ingester, feel free to suggest if you know one.

Reply:
This seems reasonable - 1 upload to the container, then another pass through the ingester.